docs/operation/Recover_From_Etcd_Permanent_Quorum_Loss.md

```diff
@@ -0,0 +1,83 @@
+## Quorum loss in ETCD Cluster
+[Quorum loss](https://etcd.io/docs/v3.3/op-guide/recovery/) means when majority of ETCD pods(Greater than or equal to n/2 + 1) are down simultaneously for some reason. 
+
+There are two types of quorum loss that can happen to ETCD multinode cluster -
+
+1. **Transient quorum loss** - A quorum loss is called transient when majority of ETCD pods are down simultaneously for some time. The pods may be down for network unavailability, high resource usages etc. When the pods come back after some time, they can re-join to the cluster and the quorum is recovered automatically without any manual intervention. The data directory must not be corrupted for majority of pods in this type of quorum loss.
+
+2. **Permanent quorum loss** - A quorum loss is called permanent when majority of ETCD pods are down simultaneously due to unavailability of data volumes that are/were attached to the ETCD pods to hold the ETCD data. If majority of ETCD data volumes are unavailable **SIMULTANEOUSLY**, the etcd cluster is not going to recover automatically from the quorum loss. A human operator will now need to intervene and execute the following steps to start a fresh multi-node ETCD cluster.
+
+## ETCD clusters in shoot control plane
+If permanent quorum loss occurs to a [Multinode ETCD cluster](https://github.com/gardener/etcd-druid/tree/master/docs/proposals/multi-node), the operator needs to note down the PVCs, configmaps, statefulsets, CRs etc related to that ETCD cluster and work on those resources only. Following steps guide a human operator to recover from permanent quorum loss of a ETCD cluster. We assume the name of the ETCD CR for the ETCD cluster is `etcd-main`
+
+   1. Add the following annotation to the etcd object `k annotate etcd etcd-main druid.gardener.cloud/ignore-reconciliation="true"`
+   3. Note down the configmap name that is attached to the `etcd-main` statefulset. If you describe the statefulset with `k describe sts etcd-main`, look for the lines similar to following lines to identify attached configmap name. It will be needed at later stages:
```

Comment:
```suggestion
   3. Note down the configmap name that is attached to the `etcd-main` statefulset. If you describe the statefulset with `kubectl describe sts etcd-main`, look for the lines similar to following lines to identify attached configmap name. It will be needed at later stages:
```